### DISCUSSION TITLE
Command 'lets' not found, did you mean: [ERROR]

### DISCUSSION THREAD

**Discussion by mustafakendiguzel** [19/classroom#41] [Anonim] [answered]
Merhabalar hocam, puanlama sistemiyle kendi puanımı öğrenmek istediğimde lets modülünün bulunamadığı hatasını alıyorum lets modülü kullanmak şart olmasa da işlemleri çok kolaylaştırdığı için kullanmak istiyorum ama terminal üzerinden puanlama yaptıramıyorum. İnternet üzerinde "lets grad" nasıl yükleneceğine dair bir araştırma yaptım
fakat internette  sizin sitenizde ve diğer tüm forumlarda aradğımda hiçbir sonuç çıkmadı.
Ayrıca belki yüklenir diye şu komutları da denedim fakat bir sonuç vermedi.Denediklerim;
```
sudo apt install lets
sudo apt install lets-grad

```
Ayriyetten uygulamayı tekrar baştan yükleyip bu sorunların çözülüp çözülmeyeceğini denedim fakat o da işe yaramadı[Aynı sorunu yaşayan birini daha gördüm ](https://github.com/19/classroom/discussions/40)ama burdan çalışmak benim için kritik olduğu için çok uğraşıp farklı denemeler yapmama rağmen başarılı olamadım.Eğer bunun başka bir çözümü yoksa mecbur olarak tarayıcı üzerinden çalışmaya devam edeceğim

Hatanın Tam Hali
```
Command 'lets' not found, did you mean:
  command 'less' from deb less (590-1)
  command 'les' from deb atm-tools (1:2.5.1-4build1)
  command 'lecs' from deb atm-tools (1:2.5.1-4build1)
Try: sudo apt install <deb name>
```

**Comment by roktas** [comment, marked as answer]
Classroom kurulumun (bir nedenle) başarısız olmuş.  Tarayıcıdan devam et.  Ayrıca sistemin hakkında en ufak bir bilgi vermemişsin.  Ben tahminle "Classroo ortamı" lafını kullanıyorum.  Windows üzerinde Classroom mu denedin, yoksa bir Linux sistemi mi kullanıyorsun bilmiyorum.  @mustafakendiguzel

**Reply by mustafakendiguzel** [reply]
Hocam merhabalar Clasroom ortamında github ile ilişkendirdiğimiz ödevleri kontrol ederken bu sorunu yaşadım
Bilgisayarımın ;
İşletim Sistemi : Windows 10
RAM : 8,00 GB
İşlemci : Intel(R) Core(TM) i5-10210U CPU 
1.60GHz 2.11 GHz
Kurulumda bir takım sıkıntılar çıkmıştı ve tekrar yüklemiştim ve başarıyla yuklenmişti ama sanırım tam başarılı olamamış.

**Reply by roktas** [reply]
Anladım.  Dediğimi gibi kurulumun sorunlu, bir tahmin yürütmem de zor.

**Reply by mustafakendiguzel** [reply]
Tamamdır hocam,web üzerinden devam edeceğim zamanınızı ayırdığınız için teşekkür ederim.
